### DISCUSSION TITLE
Twit reminder app

### DISCUSSION THREAD

**Discussion by ramyareye** [WeAreMahsaAmini/Core#133] [Ideas]
Hello,

We can have a reminder app for doing twits and retwits. 
It's simply possible to handle this routine by setup reminders in phone but it can be hard to ask everyone to do it.

Features:
- [ ] Set reminders (notifications) with configurable intervals
- [ ] Can be muted for a while
- [ ] iOS and Android
- [ ] Mac 
- [ ] Windows
- [ ] Suggested Hashtags
- [ ] trending / important twits 
- [ ] fake hashtags!

Please share your thoughts and ideas!

**Comment by kaveh6** [comment]
if the purpose is only having a reminder : I think instead of developing an app , we can prepare a doc or a short video since most phones have this functionalities build-in . 
if the app could also tweet on behalf of the person and send some pre-written tweets , that's not too bad . it's only a matter of getting people to install it .
